nano/node/common.cpp

```diff
@@ -1221,6 +1256,15 @@ void nano::telemetry_ack::serialize (nano::stream & stream_a) const
 	}
 }
 
+std::string nano::telemetry_ack::to_string () const
+{
+	std::stringstream stream;
+
```

Comment:
could be a one-liner

Reply:
are you thinking just drop the stringstream object entirely? I'm only using it because I saw it was used in a separate to_string function. Is it only useful if you're formatting multiple variables together or something?

Reply:
I also made telemetry_req::to_string() a one liner. It was very similar

Reply:
string stream is a flexible stream buffer that can accumulate lots of data in a flexible way and you can output formatting rules to format your data. It is convenient.
But in this case, your data is ready, no processing needed, just pass it along as it is.